src/tracker.rs

```diff
@@ -48,6 +63,26 @@ impl Tracker {
         }
     }
 
+    /// The `namespace` of this [Tracker]
+    pub fn get_namespace(&self) -> &str {
```

Comment:
Nit: not sure about this, but I found some Rust conventions that state the getters should not contain the `get_` part, so only `fn namespace(&self) -> &str`, [see here](https://github.com/rust-lang/rfcs/blob/master/text/0344-conventions-galore.md#gettersetter-apis).

I don't really have a strong preference, both options sound good to me. But if we use `subject_mut` for getting the mutable reference, it makes sense to me to have a `subject` getter for immutable reference.

Anyway, up to your consideration, but if you decide to change this, please do so also in all other places where we have getters (e.g., for `get_subject` in event types).

Reply:
Convention sounds like a good idea to follow to me. I've changed them in tests too for consistency.